Talk roles

edpaget opened the pull request:
Closes #373

aweiksnar commented:
This looks awesome! 
I'll definitely use user-search.cjsx for email style messaging in the inbox page as well

aweiksnar commented on css/user-search.styl:
```diff
@@ -0,0 +1 @@
+@import "default.css"
```
I think it may be worth changing up this line to `@import "../node_modules/react-select/dist/default.css"` and then the css import lines in the shell scripts can be just `--include-css  \` so the css is all bundled in the same file

edpaget commented:
Cool I changed the CSS and added a debounce to the search function. The wait time might need a little tweaking but the 200ms felt right when I was testing it.

aweiksnar commented:
Sweet looks great!
